workload-metrics/manifests/workload-metrics-deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: workload-metrics-example
+  name: workload-metrics-example
+  namespace: gke-workload-metrics
+spec:
+  selector:
+    matchLabels:
+      app: workload-metrics-example
+  template:
+    metadata:
+      labels:
+        app: workload-metrics-example
+    spec:
+      containers:
+      - image: nilebox/prometheus-example-app@sha256:dab60d038c5d6915af5bcbe5f0279a22b95a8c8be254153e22d7cd81b21b84c5
```

Comment:
where does the code for this live? is there an image tag you could use here instead of a SHA?

Reply:
I actually moved all of the code into this repo now and included a cloudbuild.yaml to tag a 1.0 image and push to google-samples